app/views/IndexPage.scala.html

```diff
@@ -52,7 +52,7 @@ <h2>Blog Posts</h2>
                         @blogs.map { blog =>
                             <tr>
                                 <td>@{blog.createdDate.displayFormat}</td>
-                                <td><a href="@{blog.url}">@{blog.title}</a></td>
+                                <td><a id="link-to-blog-post" href="@{blog.url}">@{blog.title}</a></td>
```

Comment:
This will create multiple <a .../> with the same id each blog post will need its own id"